soundata/datasets/dcase_bioacoustic.py

```diff
@@ -0,0 +1,628 @@
+    Number of audio recordings        |    1
+    Total duration                    |    10 mins
+    Total classes                    |    1
+    Total events                    |    357
+    Ratio event/duration            |    0.06
+    Sampling rate                    |    22050 Hz
+    ====================================================
+    TRAINING SET/MT
+    ====================================================
+    Number of audio recordings        |    2
+    Total duration                    |    1 hour and 10 mins
+    Total classes                    |    4
+    Total events                    |    1294
+    Ratio event/duration            |    0.04
+    Sampling rate                    |    8000 Hz
+    -----------------------------------------------------
+    TRAINING SET/WMW
+    -----------------------------------------------------
+    Number of audio recordings        |    161
+    Total duration                    |    4 hours and 40 mins
+    Total classes                    |    26
+    Total events                    |    2941
+    Ratio event/duration            |    0.24
+    Sampling rate                    |    various sampling rates
+    -----------------------------------------------------
+
+    -----------------------------------------------------
+    VALIDATION SET
+    -----------------------------------------------------
+    Number of audio recordings        |    18
+    Total duration                    |    5 hours and 57 minutes
+    Total classes                    |    5
+    Total events                     |    1077
+    -----------------------------------------------------
+    VALIDATION SET/HB
+    -----------------------------------------------------
+    Number of audio recordings        |    10
+    Total duration                    |    2 hours and 38 minutes
+    Total classes                     |    1
+    Total events                     |    712
+    Ratio event/duration            |    0.7
+    Sampling rate                    |    44100 Hz
+    -----------------------------------------------------
+    VALIDATION SET/PB
+    -----------------------------------------------------
+    Number of audio recordings        |    6
+    Total duration                    |    3 hours
+    Total classes                    |    2
+    Total events                     |    292
+    Ratio event/duration            |    0.003
+    Sampling rate                    |    44100 Hz
+    -----------------------------------------------------
+    VALIDATION SET/ME
+    -----------------------------------------------------
+    Number of audio recordings        |    2
+    Total duration                    |    20 minutes
+    Total classes                    |    2
+    Total events                     |    73
+    Ratio event/duration            |    0.01
+    Sampling rate                    |    44100 Hz
+    -----------------------------------------------------
```

Comment:
This table is not rendering properly in the docs! Also, is this table somewhere else on a website? Maybe adding it here is too much, and we could summarize it here while linking to the web where the table is. How does that sound?

Reply:
removed it